apps/frontend/src/app/3_organisms/ZeroLocForm/Label.tsx

```diff
@@ -25,41 +22,29 @@ export const Label: FC<CustomLabelProps> = ({
   maxAmount,
   symbol,
   tabs,
-  activeTab,
   onTabChange,
   onMaxAmountClicked,
   hasTrove = false,
 }) => {
-  const handleTabChange = useCallback(
-    (value: AmountType) => () => onTabChange(value),
-    [onTabChange],
+  const [index, setIndex] = useState(0);
+
+  const onChangeIndex = useCallback(
+    (index: number) => {
+      setIndex(index);
+      onTabChange(tabs[index].amountType);
+    },
+    [onTabChange, tabs],
   );
 
   return (
     <div className="w-full flex flex-row justify-between gap-4 items-center">
-      {hasTrove ? (
-        <div className="flex flex-row items-center justify-start gap-2">
-          {tabs.map(tab => (
-            <button
-              key={tab.value}
-              className={classNames(
-                'font-roboto font-semibold text-[11px] px-3 py-1 rounded bg-gray-80 whitespace-nowrap',
-                {
-                  'text-gray-30 text-opacity-75':
-                    !tab.disabled && tab.value !== activeTab,
-                  'text-gray-40 cursor-not-allowed': tab.disabled,
-                  'bg-gray-70 text-primary-20': tab.value === activeTab,
-                },
-              )}
-              onClick={handleTabChange(tab.value)}
-              disabled={tab.disabled}
-            >
-              {tab.label}
-            </button>
-          ))}
-        </div>
-      ) : (
-        <>{tabs[0].label}</>
+      {hasTrove && (
+        <Tabs
+          items={tabs}
+          index={index}
+          onChange={onChangeIndex}
+          type={TabType.secondary}
+        />
```

Comment:
It works for Adjust LoC modal, but now there is no label at Open LoC modal.
Use something like:
`{hasTrove ? <Tabs .../> : <>{tabs[0].label}</>`
to make sure default label is showing up for Open LoC

Reply:
yep, good catch